WowCharComparerWebApp/Controllers/User/LoginController.cs

```diff
@@ -32,21 +46,15 @@ public IActionResult PasswordResetConfirmationView(string userID)
         [HttpPost]
         public IActionResult PasswordChanged(string newPassword, string newPasswordConfirmation, Guid userID)
         {
-            List<string> passwordsToCheck = new List<string>();
-            {
-                passwordsToCheck.Add(newPassword);
-                passwordsToCheck.Add(newPasswordConfirmation);
-            }
+            List<(bool, string)> checkedPasswords = new List<(bool, string)>();
 
-            List<(bool,string)> checkedPasswords = new List<(bool,string)>();
-            RegisterController registerController = new RegisterController();
-
-            for (int index = 0; index < passwordsToCheck.Count(); index++)
-                checkedPasswords.AddRange(registerController.CheckPassword(passwordsToCheck[index]));
+            checkedPasswords.AddRange(_passwordValidationManager.CheckPassword(newPassword));
+            checkedPasswords.AddRange(_passwordValidationManager.CheckPassword(newPasswordConfirmation));
+            checkedPasswords.Add((newPassword.Equals(newPasswordConfirmation), UserMessages.UserConfirmPasswordNoMatch));
```

Comment:
Seems like you want to add true when password is not correct.

Reply:
Thanks, fixed.